echo/EchoTest.java

```diff
@@ -1 +1,22 @@
 package echo;
+
+import org.junit.Test;
+import static org.junit.Assert.assertTrue;
+import java.io.*;
+
+import echo.Echo;
+
+public class EchoTest {
+
+  private final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+
+  @Test
+  public void asksForInput() {
+    System.setOut(new PrintStream(outContent));
```

Comment:
JUnit has an annotation `@Before` where you can put setup, and is run before every test case in the file. You might find it tidies up the test a little, leaving the test containing the 'important' parts.